test/git-open.bats

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/env bats
+
+load "test_helper/bats-support/load"
+load "test_helper/bats-assert/load"
+
+foldername="sandboxrepo"
+
+setup() {
+  create_git_sandbox
+}
+
+
+##
+## GitHub
+##
+
+@test "gh basic" {
+  git remote set-url origin "git@github.com:user/repo.git"
+  git checkout -B "master"
+  run ../git-open
+  assert_output "https://github.com/user/repo/"
+}
+
+@test "gh branch" {
+  git remote set-url origin "git@github.com:user/repo.git"
+  git checkout -B "mybranch"
+  run ../git-open
+  assert_output "https://github.com/user/repo/tree/mybranch"
+}
+
+@test "gh non-origin remote" {
+  git remote set-url origin "git@github.com:user/repo.git"
+  git remote add upstream "git@github.com:upstreamorg/repo.git"
+  run ../git-open "upstream"
+  assert_output "https://github.com/upstreamorg/repo/"
+
+  git checkout -B "mybranch"
+  run ../git-open "upstream" "otherbranch"
+  assert_output "https://github.com/upstreamorg/repo/tree/otherbranch"
+}
+
+@test "gh without git user" {
+  # https://github.com/paulirish/git-open/pull/63
+  git remote set-url origin "github.com:paulirish/git-open.git"
+  run ../git-open
+  assert_output "https://github.com/paulirish/git-open/"
+}
+
+@test "gh ssh origin" {
+  git remote set-url origin "ssh://git@github.com/user/repo"
+  run ../git-open
+  assert_output "https://github.com/user/repo/"
+
+  # https://github.com/paulirish/git-open/pull/30
+  git remote set-url origin "ssh://git@github.com/user/repo.git"
+  run ../git-open
+  assert_output "https://github.com/user/repo/"
+}
+
+@test "gh: git open issue" {
```

Comment:
thx. i added colons to the rest of them:

![image](https://user-images.githubusercontent.com/39191/27247085-a9d88508-52aa-11e7-80ff-8b70dda10215.png)